### ISSUE TITLE
[Dashboard Feature] 

### ISSUE THREAD

**Issue by corinamarkodimitraki** [rodekruis/cashpo#207]
Add MHT to dashboard as new location

**Comment by angelinaRC** [comment]
![image](https://user-images.githubusercontent.com/62642010/210733599-cc504116-8788-46a2-a33d-3946c53c794c.png)


I see MHT locations there, or should it be just MHT?

**Comment by corinamarkodimitraki** [comment]
Indeed @angelinaRC . These were renamed to Korinthos clinics, Lesvos clinics etc. Next to that, you also have a Mobile health Team. It would be great to update the names of that list.

In the screenshot you can see all the locations that are now available on Espo, with the correct way of writing them. Is it possible to update the list in PowerBI to these? 

![Screenshot 2023-01-05 at 09 21 56](https://user-images.githubusercontent.com/60880112/210734404-5799b724-adfc-44c6-bdbc-9189aacb5491.png)

**Comment by angelinaRC** [comment]
@corinamarkodimitraki can you send it as a text (so that I can copy paste it directly)?

**Comment by corinamarkodimitraki** [comment]
EHS Ano Liosion |  
-- | --


EHS Ano Liosion
EHS Athens
EHS Kallitheas
Mobile Health Team
Korinthos Clinics
Lesvos Clinics
Malakasa Clinics
Nea Kavala Clinics
Ritsona Clinics
Serres Clinics

**Comment by angelinaRC** [comment]
one more question - so all the MHT + something are now just MHT?

![image](https://user-images.githubusercontent.com/62642010/210741843-104dbd18-8c2f-4711-a1a4-2c18d325f812.png)

![image](https://user-images.githubusercontent.com/62642010/210741964-38524097-fd7a-497a-a85b-bc14e8a7d77f.png)

(also, there is a vaccination case that has weirdly two locations, is that normal?)

**Comment by corinamarkodimitraki** [comment]
No, the list I sent you is how it should appear. So for example, MHT Korinthos becomes now Korinthos clinics. In addition, now we have added the Mobile Health Team as a location. I hope this is clear?

I checked the vaccination you mention and removed the wrong location. Once the dashboard renews, it should show only the Mobile Health Team.

**Comment by corinamarkodimitraki** [comment]
The HRC code for people registered under Mobile Health Team starts with ΚΟΥ

**Comment by corinamarkodimitraki** [comment]
[13:22] Markodimitraki, Corina
Korinthos is 62e7a10790330b078

[13:22] Markodimitraki, Corina
Lesvos: 62cff99f4f545e496

[13:22] Markodimitraki, Corina
Malakasa: 6384997792778cc7c

[13:23] Markodimitraki, Corina
Mobile health team: 637f593f3a762de33

[13:23] Markodimitraki, Corina
Nea Kavala: 638499c380486181b

[13:23] Markodimitraki, Corina
Ritsona: 638499ac48af6b0ce

[13:23] Markodimitraki, Corina
Serres: 638499828ed709ee3
